multiversx/mx-chain-go
# Time between heartbeats at bootstrap

## Reasoning behind the pull request
- during bootstrap, heartbeats will be used to keep track of trie sync progress. Currently, the time between sends is the same as regular heartbeats, 5 min, which may be too much for a proper progress bar
  
## Proposed changes
- added new config value

## Testing procedure
- start a node on testnet. Either check logs for `heartbeat message sent` to be ~1 min, either check https://testnet-gateway.multiversx.com/node/heartbeatstatus and make sure the `numTrieNodesReceived` for your node increases each ~minute during **trie sync**
- after bootstrap, the same log should be after ~5min, and `numTrieNodesReceived` 0

## Pre-requisites

Based on the [Contributing Guidelines](https://github.com/multiversx/mx-chain-go/blob/master/.github/CONTRIBUTING.md#branches-management) the PR author and the reviewers must check the following requirements are met:
- was the PR targeted to the correct branch?
- if this is a larger feature that probably needs more than one PR, is there a `feat` branch created?
- if this is a `feat` branch merging, do all satellite projects have a proper tag inside `go.mod`?
